# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../net-tools/fddi-hotfix-2.6.18.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/lib/fddi.c.orig	2000-03-05 12:26:02.000000000 +0100
+++ net-tools-1.60/lib/fddi.c	2006-09-24 00:58:02.000000000 +0200
@@ -17,6 +17,7 @@
 #include "config.h"
 
 #include <features.h>
+#include <asm/byteorder.h> 
 
 #if HAVE_HWFDDI
 #include <sys/types.h>
